### ISSUE TITLE
Support for Daydream VR?

### ISSUE THREAD

**Issue by Claytonious** [playgameservices/play-games-plugin-for-unity#1941]
Is there yet any way to successfully use this plugin on Daydream games without being rejected? More background information here: https://forum.unity.com/threads/google-play-games-integration.450250/.

Thanks!

**Comment by Claytonious** [comment]
Oh, wow - I now noticed https://github.com/playgameservices/play-games-plugin-for-unity/blob/6bc438114dd56d000782655a8bb4ee38ae40a1c5/source/PluginDev/Assets/GooglePlayGames/BasicApi/PlayGamesClientConfiguration.cs#L279. Does anybody know if this works?

**Comment by JimmyDeemo** [comment]
Little bit of into in #1943 about this. Sorry I didn't see this post.

**Comment by Dirly** [comment]
I tried what was suggested in #1943 but it just doesn't seem like i can initialize the config.  This does seem to make sense though: 
`
 public Builder EnableHidePopups() `

**Comment by JimmyDeemo** [comment]
Sorry @Dirly, I'm afraid I didn't try the gravity value. I have not found a solution to this other than to set up a system where you remember the sign in status of the user and sign them out/in again where appropriate.

**Comment by Dirly** [comment]
@JimmyDeemo You mean to silently log in? I kind am already doing that but it seems as though there is still a small welcome popup which appears in the top middle of the screen.

**Comment by JimmyDeemo** [comment]
Yes, the only reason it works for us is that we use VR for a certain feature and then return to the main game.

**Comment by claywilkinson** [comment]
The EnableHidePopups works to get rid of the "splash" screen that Play Games Services displays when signing in and the notifications when unlocking achievements, etc.

Unfortunately, there is not a solution for signing in without using 2d UI.  For that you need to exit VR, sign in, then re-enter VR.

Another alternative it to write the 2D code in a separate Activity using Java, then once sign-in (and any other 2d things) are done, launch the Unity activity.

**Comment by Dirly** [comment]
@claywilkinson 
The EnableHidePopups does work with android 7.1.1  but it doesn't actually work with 8.0.0.  Authentication fails, its really odd.   This is happening on latest GVR and GPGS unity plugins.

    private MultiplayerManager(){  /*Plays on awake*/
        PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder()
            .EnableSavedGames()
            //.EnableHidePopups()  /*if put in will break auth*/
        .Build();
        PlayGamesPlatform.InitializeInstance(config);
        PlayGamesPlatform.DebugLogEnabled = true;
    }
    public void Activate(){  /* Fires on start */
        PlayGamesPlatform.Activate();
    }

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
